package.json

```diff
@@ -97,7 +97,8 @@
     "jest-silent-reporter": "^0.5.0",
     "nodemon": "^2.0.16",
     "nyc": "^14.1.1",
-    "prettier": "^2.8.8",
+    "prettier": "^3.3.3",
```

Comment:
I had to upgrade prettier to latest (above 3.x) because the JSON sort plugin has this requirement.